packages/docs/src/en/directives/transition.md

```diff
@@ -81,12 +81,22 @@ The above example will delay the transition and in and out of the element by 50
 
 By default, Alpine's `x-transition` applies both a scale and opacity transition to achieve a "fade" effect.
 
-If you wish to only apply the opacity transition (no scale), you can accomplish that like so:
+If you wish to only apply an opacity transition (no scale), you can accomplish that like so:
 
 ```alpine
 <div ... x-transition.opacity>
 ```
 
+To customize the opacity transition properties, you'd use the same modifiers as above, but preceded with `.opacity`, like so:
+
+```alpine
+<div ...
+    x-transition.opacity
+    x-transition.opacity.duration.500ms
+    x-transition.opacity.delay.50ms
```

Comment:
I'd worry this implies multiple x-transition is allowed.

Reply:
I might be missing some understanding, as I thought that you could do this:

```
x-transition
x-transition.scale.80
```

To keep the scale + opacity transition, but customize the scale of the scale transition.

Having tried this, now I see you can't.

How could that be done, keep the opacity+scale, but customize one or both of them?

Reply:
`x-transition.opacity.scale.80` probably...